src/NexusMods.App.UI/Pages/LibraryPage/LibraryComponents.cs

```diff
@@ -121,6 +121,7 @@ public InstallAction(
 
             CommandInstall = isInstalled.Value
                 .Select(static isInstalled => !isInstalled)
+                .ObserveOnUIThreadDispatcher()
```

Comment:
This should be removed, `ValueComponent` should guarantee that updates happen on the UI thread. It currently doesn't in this case, but we should rather fix that issue than putting `OnUI` here.